### PR TITLE
Add created_at to events

### PR DESCRIPTION
**Pull request by codetheweb**
See https://github.com/seamapi/seam-connect/issues/1938

### CODE REVIEW

File: src/events.ts [hunk 1]
```diff
@@ -119,7 +121,9 @@ export type SeamEvent =
   | FailedToSetOnDeviceAccessCodeEvent
   | DelayInSettingOnDeviceAccessCodeEvent
   | NoiseDetectedEvent
-  | ConnectedAccountConnected
-  | ConnectedAccountDisconnected
+  // | ConnectedAccountConnected
+  // | ConnectedAccountDisconnected
```
Comment (seveibar) [inline src/events.ts:125]: intentionally commented out?

File: src/events.ts [hunk 2]
```diff
@@ -119,7 +121,9 @@ export type SeamEvent =
...
   | LockLockedEvent
   | LockUnlockedEvent
+
+type a = Extract<SeamEvent, { event_type: "device.connected" }>["payload"]
```
Comment (seveibar) [inline src/events.ts:129]: hmm ghost code?

File: src/events.ts (hunk 1 above)
Comment (codetheweb) [reply src/events.ts:125]: whoops I have no idea how that snuck in, thanks

**Review by seveibar — APPROVED** [review]

File: src/events.ts (hunk 2 above)
Comment (codetheweb) [reply src/events.ts]: not my finest pr 😅

Comment (seveibar) [reply src/events.ts]: HUMP DAY 😂

**Pull request merged**